core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -94,10 +98,15 @@ object ToolUtils extends Logging {
               // Normally it should not happen that invocation target is null.
               logError(s"Unknown exception while parsing an event", i)
             }
-          case j: com.fasterxml.jackson.core.JsonParseException =>
+          case j: com.fasterxml.jackson.core.io.JsonEOFException =>
```

Comment:
nit: Can we keep consistent case variables?

Earlier we had, `case j: com.fasterxml.jackson.core.JsonParseException =>` in both cases.

In this change, we have: `case j: com.fasterxml.jackson.core.JsonParseException =>` and `case k: com.fasterxml.jackson.core.JsonParseException =>`

Reply:
Done